src/test/java/neworangehrm/com/opensourcehrm/steps/CommonSteps.java

```diff
@@ -4,58 +4,82 @@
+import neworangehrm.com.opensourcehrm.util.PageUtil;
 
-import static neworangehrm.com.opensourcehrm.pages.LoginPage.*;
-import static neworangehrm.com.opensourcehrm.util.DataUtil.*;
 import static neworangehrm.com.opensourcehrm.util.DataUtil.baseUrl;
 
 public class CommonSteps implements En {
 
-    DriverUtil driverUtil;
+    DriverUtil driverUtil = new DriverUtil();
+    WebDriver driver = driverUtil.getDriverInstance();
+    PageUtil pageUtil = new PageUtil();
+    ReadTestCases readTestCases;
 
     @Before
     public void setUp() {
 
         driverUtil = new DriverUtil();
     }
     public CommonSteps() {
-        Given("^I am on the (.*) page$", (String webPage) -> {
+        Given("^I am on the \"([^\"]*)\" page$", (String webPage) -> {
             switch(webPage.toLowerCase()){
                 case "login":
                     driverUtil.getDriverInstance().get(baseUrl);
+                    pageUtil.setCurrentPage("login");
                     break;
                 case "dashboard":
-                    driverUtil.getDriverInstance().get(baseUrl);
-                    driverUtil.getDriverInstance().findElement(getElementLocator("username")).sendKeys("admin");
-                    driverUtil.getDriverInstance().findElement(getElementLocator("password")).sendKeys("admin");
-                    driverUtil.getDriverInstance().findElement(getElementLocator("Login")).click();
+                    //driverUtil.getDriverInstance().get(baseUrl);
+                    pageUtil.setCurrentPage("dashboard");
+                    //pageActions.login(driver, "Verify valid admin login");
+//                    driverUtil.getDriverInstance().findElement(getElementLocator("username")).sendKeys("admin");
+//                    driverUtil.getDriverInstance().findElement(getElementLocator("password")).sendKeys("admin");
+//                    driverUtil.getDriverInstance().findElement(getElementLocator("Login")).click();
+                    break;
+                case "usermanagement":
+                    pageUtil.setCurrentPage("usermanagement");
                     break;
                 default:
                     driverUtil.getDriverInstance().get(baseUrl);
             }
 
         });
 
-        When("^I fill the login form and submit$", () -> {
-            driverUtil.getDriverInstance().findElement(getElementLocator("username")).sendKeys("admin");
-            driverUtil.getDriverInstance().findElement(getElementLocator("password")).sendKeys("admin");
-            driverUtil.getDriverInstance().findElement(getElementLocator("Login")).click();
+        When("^I fill the login form and submit for \"([^\"]*)\"$", (String testCaseName) -> {
+            ((LoginPage) pageUtil.getCurrentPage()).login(driver, testCaseName);
+            //pageActions.login(driver, "Verify valid admin login");
+//            driverUtil.getDriverInstance().findElement(getElementLocator("username")).sendKeys("admin");
```

Comment:
no code should be in comments.

Reply:
Ok.
Deleted all commented unwanted code, now we can baseline the version as 1.0